.travis.yml

```diff
@@ -37,8 +37,8 @@ install:
   - ./build/cmake/bin/cmake .
   - .travis/install_golang_tools.sh
 script:
-  - make lint -j
-  - make travis
+  - travis_wait 30 make lint -j
```

Comment:
curious what is wait for?

Reply:
updated with new method, this intends to mitigate timeout issue

Reply:
might need to check how much slow down in the c code compilation

Reply:
yeah, had a ticket created to improve it, most time is on the scratch_transform and dimension_transform, as we added output types from 3 to 5.